web3j-ext/web3j-ext/src/main/java/org/web3j/crypto/KlayTransactionEncoder.java

```diff
@@ -44,6 +44,15 @@
  */
 public class KlayTransactionEncoder extends TransactionEncoder{
 
+    public static byte[] signMessage(RawTransaction rawTransaction, long chainId, KlayCredentials credentials) {
```

Comment:
Why we need the **chainId** in **signMessage**? 
I think we need to fill all the fields before push a tx to the **signMessage**